zcash_client_backend/src/data_api.rs

```diff
@@ -1744,14 +1744,14 @@ pub trait WalletWrite: WalletRead {
         received_tx: DecryptedTransaction<Self::AccountId>,
     ) -> Result<(), Self::Error>;
 
-    /// Saves information about a transaction constructed by the wallet to the persistent
-    /// wallet store.
+    /// Saves information about transactions constructed by the wallet to the persistent
+    /// wallet store. This must be called before the transactions are sent to the network.
```

Comment:
The new sentence doesn't need to be in the summary paragraph:
```suggestion
    /// wallet store.
    ///
    /// This must be called before the transactions are sent to the network.
```